cmd/clusterctl/cmd/root.go

```diff
@@ -118,6 +123,19 @@ func init() {
 	RootCmd.PersistentFlags().StringVar(&cfgFile, "config", "",
 		"Path to clusterctl configuration (default is `$HOME/.cluster-api/clusterctl.yaml`) or to a remote location (i.e. https://example.com/clusterctl.yaml)")
 
+	RootCmd.AddGroup(
+		&cobra.Group{
+			ID:    groupManagement,
+			Title: "Management commands:",
```

Comment:
```suggestion
			Title: "Cluster Management Commands:",
```
For consistency with kubectl